src/actions/Actions.ts

```diff
@@ -172,10 +176,14 @@ export const AsyncActions = {
     downloadPostsFromFeeds: (feeds: Feed[]): Thunk => {
         return async (dispatch, getState) => {
             const previousPosts = getState().rssPosts;
+            const feedsWithoutOnboarding = feeds.filter(feed => feed.feedUrl !== FELFELE_ASSISTANT_URL);
             // TODO this is a hack, because we don't need a feed address
             const swarm = Swarm.makeReadableApi({user: '', topic: ''}, getState().settings.swarmGatewayAddress);
-            const downloadedPosts = await loadPostsFromFeeds(swarm, feeds);
-            const posts = mergeUpdatedPosts(downloadedPosts, previousPosts);
+            const allPosts = await Promise.all([
+                loadSocialPostsFromFeeds(swarm, feedsWithoutOnboarding, dispatch),
```

Comment:
I would call this function something else. Also I wouldn't pass `dispatch` here, because we use it as `any` but a function that updates the feeds.